examples/stub.php

```diff
@@ -444,14 +441,7 @@ class Pool
      *
      * @link http://www.php.net/manual/en/pool.__construct.php
      */
-    public function __construct($size, $class, array $ctor = array()) {}
-
-    /**
-     * Shuts down all Workers, and collect all Stackables, finally destroys the Pool
-     *
-     * @link http://www.php.net/manual/en/pool.__destruct.php
-     */
-    public function __destruct() {}
+    public function __construct(int $size, ?string $class = null, ?array $ctor = array()) {}
```

Comment:
This doesn't look quite right to me. The `$class` and `$ctor` parameters should not be nullable (attempting to pass in `null` to either of them will cause ZPP to fail). I think the arg info for the constructor should be updated to reflect this as well. Also, the default class is `Worker`, according to [the implementation](https://github.com/krakjoe/pthreads/blob/2bc44069c1f3c83c866464b60345a333eacee598/classes/pool.h#L80).

Reply:
@tpunt this was generated by a script, so if this is wrong then the arginfo is likely wrong as well. Probably this is the same issue as seen in #835 .

Reply:
@dktapps I see. Would you like to submit a PR to rectify this too? If not, I'll do so later today.

Reply:
I can push a fix with #835 if you want, since it's the same subject.